src/status_im/ui/screens/chat/photos.cljs

```diff
@@ -2,18 +2,21 @@
   (:require [clojure.string :as string]
             [status-im.ui.components.react :as react]
             [status-im.ui.screens.chat.styles.photos :as style]
+            [status-im.ui.screens.profile.db :as profile.db]
             [status-im.utils.identicon :as identicon]
             [status-im.utils.image :as utils.image])
   (:require-macros [status-im.utils.views :refer [defview letsubs]]))
 
 (defn photo [photo-path {:keys [size
                                 accessibility-label]}]
-  [react/view {:style (style/photo-container size)}
-   [react/image {:source              (utils.image/source photo-path)
-                 :style               (style/photo size)
-                 :resize-mode         :cover
-                 :accessibility-label (or accessibility-label :chat-icon)}]
-   [react/view {:style (style/photo-border size)}]])
+  (let [identicon? (when photo-path (profile.db/base64-png? photo-path))]
```

Comment:
Why is this included here?

Reply:
WHAT? No idea how it got here, I'll rebase the PR without this crap.